### PR TITLE
[Merged by Bors] - add link to tracy compatibility table

### PR DESCRIPTION
**Pull request by hymm**
# Objective

- Fixes https://github.com/bevyengine/bevy/issues/5200

### CODE REVIEW

File: docs/profiling.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ The [Tracy profiling tool](https://github.com/wolfpld/tracy) is:
 
 There are binaries available for Windows, and installation / build instructions for other operating systems can be found in the [Tracy documentation PDF](https://github.com/wolfpld/tracy/releases/latest/download/tracy.pdf).
 
-It has a command line capture tool that can record the execution of graphical applications, saving it as a profile file. Tracy has a GUI to inspect these profile files. The GUI app also supports live capture, showing you in real time the trace of your app.
+It has a command line capture tool that can record the execution of graphical applications, saving it as a profile file. Tracy has a GUI to inspect these profile files. The GUI app also supports live capture, showing you in real time the trace of your app. The version of tracy must be matched to the version of tracing-tracy used in bevy. A compatibility table can be found on [crates.io](https://crates.io/crates/tracing-tracy) and the version used can be found [here](https://github.com/bevyengine/bevy/blob/latest/crates/bevy_log/Cargo.toml#L21).
```
Comment (hymm) [inline docs/profiling.md:26]: not sure if there's a better way to get the crate version used.

Comment (alice-i-cecile) [reply docs/profiling.md:26]: Yeah, I can't think of one unfortunately :(

**Review by alice-i-cecile — APPROVED** [review]

**Comment by mockersf** [comment]
Could you update the PR title so that it mentions tracy?

**Comment by james7132** [comment]
bors retry

**Pull request closed without merging**